roles/unicorn_user/tasks/main.yml

```diff
@@ -34,9 +34,3 @@
     group: "{{ unicorn_user }}"
     mode: 0644
 
-- name: Create sudoers configuration for unicorn user
-  become: true
-  template:
-    src: sudoers.j2
-    dest: "/etc/sudoers.d/{{ unicorn_user }}"
-    mode: 0440
```

Comment:
If we do this we won't be able to restart the unicorn when logging in as the unicorn user, which is something we do often.